### PR TITLE
BACKOFFICE FE/UI/Labeling Tool: Update UI/UX re-render image after up…

### PR DESCRIPTION
**Pull request by cuongho2**
Set border of image to red if status = R
After user click upload → re-render image object

### CODE REVIEW

File: src/store/labels/actionCreators.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export function updateImageDataById(
     id: string,
     newImageData: ImageData
 ): LabelsActionTypes {
+
```
Comment (choipd) [inline src/store/labels/actionCreators.ts]: [minor] Please do not edit unrelated code

File: src/store/labels/reducer.ts [hunk 2]
```diff
@@ -32,7 +32,9 @@ export function labelsReducer(
     state = initialState,
     action: LabelsActionTypes
 ): LabelsState {
+
```
Comment (choipd) [inline src/store/labels/reducer.ts]: [minor] Please do not edit unrelated code

File: src/views/EditorView/EditorContainer/EditorContainer.tsx [hunk 3]
```diff
@@ -150,7 +150,7 @@ const EditorContainer: React.FC<IProps> = ({
                 ? GENDER.MAN
                 : GENDER.WOMAN;
         if (activeImageData.guideStyles) {
-            // console.log('Guide styles = ', activeImageData.guideStyles);
+
```
Comment (choipd) [inline src/views/EditorView/EditorContainer/EditorContainer.tsx]: [minor] Please do not edit unrelated code

File: src/views/EditorView/EditorContainer/EditorContainer.tsx [hunk 4]
```diff
@@ -168,13 +168,14 @@ const EditorContainer: React.FC<IProps> = ({
                         })
                         .filter((style) => style !== 0)
                         .map((style) => {
-                            // console.log('style == ', style);
+                           // console.log('style == ', style);
```
Comment (choipd) [inline src/views/EditorView/EditorContainer/EditorContainer.tsx]: [minor] Please do not edit unrelated code

File: src/views/EditorView/EditorContainer/EditorContainer.tsx [hunk 5]
```diff
@@ -168,13 +168,14 @@ const EditorContainer: React.FC<IProps> = ({
...
                             const images = _.range(5).map((i) => {
                                 const st = _.find(FASHION_STYLE, {seq: style});
+
```
Comment (choipd) [inline src/views/EditorView/EditorContainer/EditorContainer.tsx]: [minor] Please do not edit unrelated code

File: src/views/EditorView/SideNavigationBar/ImagePreview/ImagePreview.tsx [hunk 6]
```diff
@@ -128,7 +129,7 @@ class ImagePreview extends React.Component<IProps, IState> {
         image: HTMLImageElement,
         imageData: ImageData
     ) => {
-        // console.log('saveLoadedImage: ', image, imageData);
+
```
Comment (choipd) [inline src/views/EditorView/SideNavigationBar/ImagePreview/ImagePreview.tsx]: [minor] Please do not edit unrelated code

File: src/views/EditorView/SideNavigationBar/ImagePreview/ImagePreview.tsx [hunk 7]
```diff
@@ -179,7 +180,7 @@ class ImagePreview extends React.Component<IProps, IState> {
 
     public render() {
         const {isChecked, style, onClick, imageData} = this.props;
-       
+
```
Comment (choipd) [inline src/views/EditorView/SideNavigationBar/ImagePreview/ImagePreview.tsx]: [minor] Please do not edit unrelated code

File: src/views/EditorView/SideNavigationBar/ImagesList/ImagesList.tsx [hunk 8]
```diff
@@ -91,6 +91,7 @@ class ImagesList extends React.Component<IProps, IState> {
         isVisible: boolean,
         style: React.CSSProperties
     ) => {
+
```
Comment (choipd) [inline src/views/EditorView/SideNavigationBar/ImagesList/ImagesList.tsx]: [minor] Please do not edit unrelated code

**Review by choipd — CHANGES_REQUESTED** [review]
There are some minor issues.

**Comment by cuongho2** [comment]
hi mr @choipd, I was removed unrelate code.

File: src/views/EditorView/EditorContainer/EditorContainer.tsx [hunk 9]
```diff
@@ -168,9 +168,9 @@ const EditorContainer: React.FC<IProps> = ({
                         })
                         .filter((style) => style !== 0)
                         .map((style) => {
-                            // console.log('style == ', style);
+                        // console.log('style == ', style);
```
Comment (choipd) [inline src/views/EditorView/EditorContainer/EditorContainer.tsx]: It would be nice if you kept the previous code when it is not related to this PR.

**Comment by cuongho2** [comment]
Mr @choipd , I was fixed all your mentain, Could you approve my new code.

**Comment by choipd** [comment]
@cuongho2 I've merged, Thanks

**Pull request merged**